spec/services/hyrax/admin_set_create_service_spec.rb

```diff
@@ -203,12 +205,16 @@
       let(:admin_set) { FactoryBot.build(:hyrax_admin_set) }
 
       context "when the admin_set is valid" do
+        let(:listener) { Hyrax::Specs::SpyListener.new }
         let(:permission_template) { Hyrax::PermissionTemplate.find_by(source_id: admin_set.id) }
         let(:grants) { permission_template.access_grants }
         let(:available_workflows) { [create(:workflow), create(:workflow)] }
 
         # rubocop:disable RSpec/AnyInstance
         before do
+          Hyrax.publisher.subscribe(listener)
+          allow_any_instance_of(Hyrax::Listeners::MetadataIndexListener)
+            .to receive(:on_collection_metadata_updated).and_return(available_workflows)
```

Comment:
what's going on here? the index listener is returning the workflows?

Reply:
Got to be a copy-paste error.  I'll take a closer look.